test/html_node_test.rb

```diff
@@ -0,0 +1,106 @@
+# frozen_string_literal: true
+require "test_helper"
+
+module ThemeCheck
+  class HtmlNodeTest < Minitest::Test
+    def test_markup
```

Comment:
Maybe missing test cases with Liquid wrapping attributes:

```
<img
{% if ... %}
class="banner"
{% else %}
class="no-banner"
{% endif %}
>
```

Reply:
Test passes :) The only case that wouldn't work is the same case that would break a prettier plugin:

```
{% if div %}
<div>
{% else %}
<span>
{% endif %}

{% if div %}
</div>
{% else %}
</span>
{% endif %}
```

But that was already a problem. And not one I think we can solve :/